### PR TITLE
Add TCB 62 about Prezi

### PR DESCRIPTION
**Pull request by mosabua**
Skeleton ready for review and input.

Please have a look @vincenzocassaro and let me know what to add and update. If you have any images or slide deck for the talk we can also link to it or add images and more.

### CODE REVIEW

File: _episodes/62.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Vincenzo has been working with data in all its forms, from data modeling to
+analytics and ML since he completed his masters in computer engineering in
+Italy. He is joining us from there, more specifically from Sicily, to chat with
+us about how he got into computers, learned about Trino, and ended up at Prezi
+now.
+
+## About Prezi
+
+what is Prezi
```
Comment (mosabua) [inline _episodes/62.md:66]: Is there a good standard text @vincenzocassaro to use .. or some info from your slide deck

**Comment by mosabua** [comment]
Ready I think .. can you review @colebow ?

File: _episodes/62.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  subproject maintainer for [Trino Gateway](https://trinodb.github.io/trino-gateway/).
+* [Recap blog post]({% post_url 2024-06-24-trino-fest-recap %}) from Trino Fest
+  with video recordings and slides is now available.
+* Trino Contributor Congregation [recap notes](https://github.com/trinodb/trino/wiki/Contributor-meetings#trino-contributor-congregation-14-june-2024) are also available.
+* [Trino Japan meetup](https://techplay.jp/event/944074) happened on the 10th of July.
+
+## Guest Vincenzo Cassaro
+
+Vincenzo has been working with data in all its forms, from data modeling to
+analytics and ML since he completed his masters in computer engineering in
```
Comment (colebow) [inline _episodes/62.md:63]:
```suggestion
analytics and ML, since he completed his masters in computer engineering in
```

File: _episodes/62.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+Italy. He is joining us from there, more specifically from Sicily, to chat with
+us about how he got into computers, learned about Trino, and ended up at Prezi
+now.
+
+## About Prezi
+
+[Prezi](https://prezi.com/) probably doesn't need any introduction, but just in
+case. Prezi is a popular and powerful platform to create and show engaging
```
Comment (colebow) [inline _episodes/62.md:71]:
```suggestion
case: Prezi is a popular and powerful platform to create and show engaging
```

**Review by colebow — APPROVED** [review]

File: _episodes/62.md (hunk 2 above)
Comment (mosabua) [reply _episodes/62.md]: updated

File: _episodes/62.md (hunk 3 above)
Comment (mosabua) [reply _episodes/62.md]: updated

**Pull request merged**